src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -2051,11 +2052,13 @@ namespace cryptonote
       unsigned int b = 0;
       const time_t time_boundary = now - static_cast<time_t>(seconds[n]);
       for (time_t ts: timestamps) b += ts >= time_boundary;
-      const double p = probability(b, seconds[n] / DIFFICULTY_TARGET_V2);
-      MDEBUG("blocks in the last " << seconds[n] / 60 << " minutes: " << b << " (probability " << p << ")");
+      const double dur = seconds[n] / DIFFICULTY_TARGET_V2; // duration of block generation we are measuring
+      const double p = probability(b, dur);
+      long b_min_exp = std::lround(threshold * dur); //
```

Comment:
This should be `const` and inside the `if (p < threshold)` code block since that's the only place it is used. Also there's trailing `//` characters, was there supposed to be a comment here?

Reply:
Thanks. Fixed both issues.